.github/workflows/action-test.yaml

```diff
@@ -0,0 +1,301 @@
+name: csv-to-csvw action
```

Comment:
@robons This file is used during development so that the we dont need to publish the action always during development.